modules/ibi-extra-partition-ibi-install-iso.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ibi-extra-partition-ibi-install-iso_{context}"]
+= Configuring additional partitions on the target host
+
+The installation ISO creates a partition for the `/var/lib/containers` directory as part of the image-based installation process. 
+
+You can create additional partitions by using the `coreosInstallerArgs` specification. For example, in hard disks with adequate storage, you might need an additional partition for storage options, such as {lvms-first}. 
+
+[NOTE]
+====
+The `var/lib/containers` partition requires at least 500 GB to ensure adequate disk space for precached images.
+====
+
+.Procedure
+
+. Edit the `image-based-installation-config.yaml` to configure additional partitions:
++
+.Example `image-based-installation-config.yaml` file
+[source,yaml]
+----
+apiVersion: v1beta1
+kind: ImageBasedInstallationConfig
+metadata:
+  name: example-extra-partition
+seedImage: quay.io/repo-id/seed:latest
+seedVersion: "4.17.0"
+installationDisk: /dev/vda
+pullSecret: '{"auths": ...}'
+
+#...
+
+skipDiskCleanup: true <1>
+coreosInstallerArgs: 
+   - "--save-partindex" <2>
+   - "6" <3>
+ignitionConfigOverride: |
+  {
+    "ignition": {
+      "version": "3.2.0" 
+    },
+    "storage": {
+      "disks": [
+        {
+          "device": "/dev/vda", <4>
+          "partitions": [
+            {
+              "label": "storage", <5>
+              "number": 6, <6>
+              "sizeMiB": 380000, <7>
+              "startMiB": 600000 <8>
```

Comment:
@tsorya - I made up a value for the startMiB field. Would this make sense, considering the space requirements for the other partitions? I don't really know what would be a typical value here

Reply:
looks good as for me

Reply:
How can a user know what's the proper value for the startMiB?
The user will probably need to inspect the seedImage... If so - maybe let's add the example of how it's done.